AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/ApkStore.kt

```diff
@@ -84,6 +84,34 @@ internal class ApkStore(
         }
     }
 
+    suspend fun uploadAdditionalApk(filePath: String): UploadedApk {
```

Comment:
What is the `filePath` in this context exactly?

In the other methods of ApkStore, file path was the path of the output (not super nice but thats what we relied on ).

If this is the GCSPath, we should use the GcsPath value class.